chainerio/__init__.py

```diff
@@ -84,44 +184,98 @@ def create_handler(uri: str) -> IO:
     return handler
 
 
-def isdir(file_path: str) -> bool:
+def isdir(path: str) -> bool:
+    """return whether the given path is a directory
```

Comment:
"Returns True if path is an existing directory."　Chainer's doc tend to adopt third-person singular present style (while Python doc is different)

https://docs.python.org/3/library/os.path.html#os.path.isdir

Reply:
Addressed in e0df1ba